client/state/selectors/get-site-options.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite, getSiteOption } from 'state/sites/selectors';
```

Comment:
It's a bit odd to me that we'd have `getSiteOptions` depend on `getSiteOption` and not the other way around. For our use here, it seems like it'd be fine to access the `site.options.default_post_format` property directly. Eventually it'd probably be good to change `getSiteOption` to:

```js
export const getSiteOption = ( state, siteId, optionName ) => 
	get( getSiteOptions( state, siteId ), optionName, null );
```